redbot/cogs/economy/economy.py

```diff
@@ -914,6 +915,7 @@ async def slottime(self, ctx: commands.Context, seconds: int):
 
         - `<seconds>` The new number of seconds to wait in between uses of the slot machine. Default is 5.
```

Comment:
The example above and the description of the `<seconds>` parameter here should be updated to reflect the type of input that the used converter accepts.